draft-packages/button/KaizenDraft/Button/IconButton.tsx

```diff
@@ -1,9 +1,9 @@
 import * as React from "react"
 import GenericButton, { IconButtonProps } from "./components/GenericButton"
 
-const IconButton: React.FunctionComponent<IconButtonProps> = (
-  props: IconButtonProps
-) => <GenericButton iconButton {...props} />
+const IconButton = <S extends string>(props: IconButtonProps<S>) => (
```

Comment:
I have no idea how this type declaration here works :( Maybe a Show & Tell topic?